### PR TITLE
added 233 open source collectives from opencollective.com db

### PR DESCRIPTION
**Pull request by xdamman**
Thank you for your work! 🙏

Let me know if I can help in any other way.

### CODE REVIEW

**Comment by xdamman** [comment]
FYI: I did a quick script to pull all open source projects on open collective and check their `package.name` on their `package.json`.
258 didn't have a `package.json` file on their repo (so about 50%)

**Comment by xdamman** [comment]
Huh, tests don't pass because "strings must use single quotes", but it's not proper JSON to use single quotes (my script exported the data as JSON)

**Comment by xdamman** [comment]
alright, changing double quotes to single quotes ;-)

File: index.js [hunk 1]
```diff
@@ -36,29 +36,252 @@ const authors = {
...
+  'webpack': 'https://opencollective.com/webpack',
+  'yo': 'https://opencollective.com/yeoman',
+  'levelup': 'https://opencollective.com/level',
+  'sierra-library': 'https://opencollective.com/sierra',
+  'vue': 'https://opencollective.com/vuejs',
+  'sketchpad': 'https://opencollective.com/sketchpad',
+  'slim-js': 'https://opencollective.com/slimjs',
+  'ng2-date-picker': 'https://opencollective.com/angular-datepicker',
+  'esdiscuss.org': 'https://opencollective.com/esdiscuss',
+  'undefined': 'https://opencollective.com/turf',
```
Comment (feross) [inline index.js]: This looks like an error!

I'm guessing that this is caused by their root `package.json` not having a `"name"` field. See here: https://github.com/Turfjs/turf/blob/master/package.json

I think this is actually pretty common for mono-repo projects. https://github.com/Turfjs/turf/blob/master/packages/turf/package.json

I wonder if this happened more than once in your script. If that happened, then you'd have overwritten the `'undefined'` field multiple times and many projects may have not made it into this dump. Thoughts?

Comment (xdamman) [reply index.js]: Ah good catch. Yes just remove that line. If they don’t have a name in their package.json there is no way you can do anything with it anyway. So not much we can do here.

**Comment by xdamman** [comment]
Thanks for merging!

**Comment by feross** [comment]
@xdamman Yep! Any chance you could regularly publish this list as a standalone npm package? This would make it easy to keep this list up-to-date.

For inspiration, take a look at some of the @nice-registry projects: https://github.com/nice-registry/all-the-package-repos

**Comment by xdamman** [comment]
Doesn't scale very well. I'd prefer finding a more scalable solution like the one being discussed here: https://github.com/feross/thanks/issues/2

**Pull request merged**